### PR TITLE
Add k8s.node.name and k8s.node.uid to semconv

### PR DESCRIPTION
**Pull request by alikhil**
According to the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/k8s.md#node) they should exist.

### CODE REVIEW

**Review by XSAM — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `trace.TraceFlags` is now a defined type over `byte` and `WithSampled(bool) TraceFlags` and `IsSampled() bool` methods have been added to it. (#1770)
 - The `Event` and `Link` struct types from the `go.opentelemetry.io/otel` package now include a `DroppedAttributeCount` field to record the number of attributes that were not recorded due to configured limits being reached. (#1771)
 - The Jaeger exporter now reports dropped attributes for a Span event in the exported log. (#1771)
+- Adds `k8s.node.name` and `k8s.node.uid` attribute keys according to specification. (#1789)
```
Comment (MrAlias) [inline CHANGELOG.md:37]:
```suggestion
- Adds `k8s.node.name` and `k8s.node.uid` attribute keys to the `semconv` package. (#1789)
```

File: semconv/resource.go [hunk 2]
```diff
@@ -127,6 +127,12 @@ const (
 	// set to any meaningful value within the environment. For example,
 	// GKE clusters have a name which can be used for this attribute.
 	K8SClusterNameKey = attribute.Key("k8s.cluster.name")
+	
+	// The name of the Node.
+	K8sNodeNameKey = attribute.Key("k8s.node.name")
```
Comment (MrAlias) [inline semconv/resource.go:132]:
```suggestion
	K8SNodeNameKey = attribute.Key("k8s.node.name")
```

File: semconv/resource.go [hunk 3]
```diff
@@ -127,6 +127,12 @@ const (
...
+	
+	// The UID of the Node.
+	K8sNodeUIDKey = attribute.Key("k8s.node.uid")
```
Comment (MrAlias) [inline semconv/resource.go:135]:
```suggestion
	K8SNodeUIDKey = attribute.Key("k8s.node.uid")
```

**Review by Aneurysm9 — APPROVED** [review]
LGTM assuming the suggestions from @MrAlias are applied.

**Comment by MrAlias** [comment]
Closing/reopeing to kick the CI system

**Review by MrAlias — APPROVED** [review]

File: semconv/resource.go [hunk 4]
```diff
@@ -127,6 +127,12 @@ const (
 	// set to any meaningful value within the environment. For example,
 	// GKE clusters have a name which can be used for this attribute.
 	K8SClusterNameKey = attribute.Key("k8s.cluster.name")
+	
+	// The name of the Node.
+	K8SNodeNameKey = attribute.Key("k8s.node.name")
+
```
Comment (MrAlias) [inline semconv/resource.go:133]:
```suggestion

	// The name of the Node.
	K8SNodeNameKey = attribute.Key("k8s.node.name")

```

**Pull request merged**